doc/timezone_specifiers.rdoc

```diff
@@ -0,0 +1,42 @@
+===Timezone Specifiers
+
+Certain methods in class Time accept arguments that specify timezones:
+
+- Time.at: keyword argument +in:+.
+- Time.new: positional argument +zone+ or keyword argument +in:+.
+- Time.now: keyword argument +in:+.
+- Time#localtime: positional argument +zone+.
+
+The value given with any of these must be one of the following:
+
+- A string offset from UTC in the form <tt>'+HH:MM'</tt> or <tt>-HH:MM</tt>,
+  where:
+
+  - +HH+ is the 2-digit hour in the range <tt>0..23</tt>.
+  - +MM+ is the 2-digit minute in the range <tt>0..59</tt>.
+
+  Examples:
+
+    Time.now(in: '-23:59') # => 2022-08-19 16:14:33.8572845 -2359
+    Time.now(in: '+23:59') # => 2022-08-21 16:12:38.0869585 +2359
```

Comment:
I think we should use `Time.at` instead of `Time.now` for these examples so that the user can reproduce these examples.

Reply:
Done.